js/bundle.go

```diff
@@ -330,10 +330,27 @@ func (b *Bundle) instantiate(logger logrus.FieldLogger, rt *goja.Runtime, init *
 	init.moduleVUImpl.ctx = context.Background()
 	init.moduleVUImpl.initEnv = initenv
 	init.moduleVUImpl.eventLoop = eventloop.New(init.moduleVUImpl)
+	pgm := init.programs[b.Filename.String()] // this just initializes the program
+	pgm.pgm = b.Program
+	pgm.src = b.Source
+	exports := rt.NewObject()
+	pgm.module = rt.NewObject()
+	_ = pgm.module.Set("exports", exports)
+	init.programs[b.Filename.String()] = pgm
```

Comment:
Same as the getter should ideally apply for the setter. For example, it probably makes sense to check if the `pgm` is properly initialized.